dataflow/flex-templates/pipeline_with_dependencies/README.md

```diff
@@ -0,0 +1,166 @@
+
+# Dataflow flex template: a pipeline with dependencies and a custom container image.
+
+This project is created to illustrate the following setup of a Dataflow Python pipeline:
+- The pipeline is a package that consists of [multiple files](https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/#multiple-file-dependencies).
+- The pipeline has at least one depenendency that is not provided in the default Dataflow runtime environment.
+- The workflow uses a [custom container image](https://cloud.google.com/dataflow/docs/guides/using-custom-containers) to preinstall dependencies and define the pipeline runtime environment.
+- The workflow uses a [Dataflow Flex template](https://cloud.google.com/dataflow/docs/concepts/dataflow-templates) to control the pipeline submission environment.
+- The runtime and submission environment use same set of Python dependencies and can be created in a reproducible manner.
+
+To illustrate this setup, we will use a pipepline that finds a longest word in an input file, crates a [FIGLet text banner](https://en.wikipedia.org/wiki/FIGlet) from of it using [pyfiglet](https://pypi.org/project/pyfiglet/), and outputs it in another file.
```

Comment:
```suggestion
To illustrate this setup, we use a pipeline that does the following:

1. Finds the longest word in an input file
2. Creates a [FIGLet text banner](https://en.wikipedia.org/wiki/FIGlet) from of it using [pyfiglet](https://pypi.org/project/pyfiglet/)
3. Outputs the text banner in another file
```

For the last step, I changed "it" to "text banner," but please verify that that's correct.